openedx/core/djangoapps/user_authn/config/waffle.py

```diff
@@ -37,3 +37,17 @@
     'enable_pwned_password_api',
     __name__
 )
+
+
+# .. toggle_name: REDIRECT_TO_LMS
+# .. toggle_implementation: WaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Set this to True if you want to cms-admin login from lms login.
+# .. in case of logout it will use lms logout also.
+# .. toggle_use_cases: open_edx
+# .. toggle_creation_date: 8-Feb-2022
+# .. toggle_target_removal_date: None
+# .. toggle_warnings: This temporary feature toggle does not have a target removal date.
```

Comment:
If it's a temporary toggle we should use `toggle_use_cases: temporary` and set an approximate removal date: https://github.com/openedx/edx-toggles/blob/master/docs/how_to/documenting_new_feature_toggles.rst

Reply:
For now, we are not adding this toggle as temporary. The plan is to keep this toggle in place for admin redirects.

Reply:
Removed the warning message since it isn't needed in this case.